draft-ietf-quic-transport.md

```diff
@@ -1763,9 +1763,14 @@ gap than it advertised.
 
 Clients MAY change connection ID at any time based on implementation-specific
 concerns.  For example, after a period of network inactivity NAT rebinding might
-occur when the client begins sending data again. A client might wish to reduce
-linkability by employing a new connection ID when sending traffic after a period
-of inactivity.
+occur when the client begins sending data again.
+
+A client might wish to reduce linkability by employing a new connection ID and
+source UDP port when sending traffic after a period of inactivity.  Changing the
```

Comment:
This may have a performance impact if the server interprets this is a full migration and not just a NAT rebind.  I think it's worth noting that as well as that this may cause the server to perform path validation before sending any significant data?